planning/behavior_path_planner/include/behavior_path_planner/scene_module/avoidance/avoidance_module_data.hpp

```diff
@@ -187,6 +187,9 @@ struct ObjectData  // avoidance target
   // longitudinal position of the CoM, in Frenet coordinate from ego-pose
   double longitudinal;
 
+  // longitudinal length of vehicle, in Frenet coordinate
+  double length;
```

Comment:
I think the variable might be a little bit unclear to future readers, since it might be confuse with ArcCoordinate or Freenet's length metric. 


i assume the name `segment_length` might explain a little bit better.

Note: but mathematician might have a much suitable term for this i guess?

Reply:
@zulfaqar-azmi-t4 
> I think the variable might be a little bit unclear to future readers, since it might be confuse with ArcCoordinate or Freenet's length metric.

"// longitudinal length of vehicle, in Frenet coordinate" is written above.

> length_segment
I don't see any differences from variable name length. it's can be simple like other variables